packages/api/models/Disease.js

```diff
@@ -18,7 +18,7 @@ const Disease = new Entity({
     omimId: 'string',
     causalGene: 'string',
     mutationImpact: 'string',
-    proteinSize: 'number',
+    proteinSize: 'string',
```

Comment:
Why the change?

Reply:
These are the possible values: 
<img width="453" alt="Screen Shot 2021-04-27 at 12 01 35 AM" src="https://user-images.githubusercontent.com/5354875/116143081-bd657380-a6eb-11eb-9d7d-c6ba151ac385.png">